scripts/surface_charge/surface_charge_calculator.py

```diff
@@ -29,30 +28,107 @@ def __init__(self, crystal, use_existing_charges: bool = False):
         self.crystal = crystal
         self.surface = None
         self._surface_charge = None
+        self.surface_atom_charge = np.nan
+        self.node_projected_charge = np.nan
+        self.node_representative_charge = np.nan
```

Comment:
what's the difference from initiating with "None"?

Reply:
As far as I am aware, numpy handles `np.nan` much better than None when you do an operation like `np.mean`